source/atom.xml

```diff
@@ -0,0 +1,35 @@
+---
+permalink: atom.xml
+use:
+- authors
+- posts
+---
+<?xml version="1.0" encoding="utf-8"?>
+<feed xmlns="http://www.w3.org/2005/Atom">
+    <title>PHP-FIG Blog Feed</title>
+    <link href="{{ site.host }}/atom.xml" rel="self"/>
+    <link href="{{ site.host }}/"/>
+    <author>
+        <name>PHP-FIG</name>
+    </author>
+    <updated>{{ site.calculated_date | date('c') }}</updated>
+    <id>{{ site.host }}/</id>
+
+    {% for post in data.posts %}
+        <entry>
+            <title type="html"><![CDATA[{{ post.title }}]]></title>
+            <link href="{{ site.host }}{{ post.url }}"/>
+            <updated>{{ post.date|date('c') }}</updated>
+            <id>{{ site.host }}{{ post.url }}</id>
+            <content type="html"><![CDATA[{{ post.blocks.content|raw }}]]></content>
```

Comment:
Couldn't you just use `striptags` instead of `raw` to cleanup all the post data? https://twig.symfony.com/doc/2.x/filters/striptags.html

Reply:
Comments below